src/css/blocks/product-card.css

```diff
@@ -0,0 +1,122 @@
+.card {
+  display: flex;
+  flex-direction: column;
+
+  font-family: Roboto, sans-serif;
+  font-weight: 400;
+  color: var(--main-color);
+
+  box-sizing: border-box;
+  max-width: 200px;
+  padding: 32px 16px 16px;
+
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+}
+
+.card__main-picture-wrapper {
+  max-width: 160px;
+  margin: 0 auto 40px;
+}
+
+.card__main-picture {
+  width: 100%;
+}
+
+.card__main-title {
+  font-size: 12px;
+  line-height: 18px;
+  font-weight: 500;
+  margin-bottom: 4px;
+}
+
+.card__item-number {
+  font-size: 10px;
+  line-height: 14px;
+  margin-bottom: 16px;
+  color: var(--secondary-color);
+}
+
+/* Star rating */
+.stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  margin-bottom: 24px;
+}
+
+.stars__list {
+  display: flex;
+}
+
+.stars__star {
+  background-repeat: no-repeat;
+  background-position: center;
+  background-image: url(../../images/star-active.svg);
+  height: 16px;
+  width: 16px;
+  cursor: pointer;
+}
+
+.stars__star:not(:first-of-type) {
+  margin-right: 4px;
+}
```

Comment:
rerun your test one more time, you may've ran them without this margin:
<img width="779" alt="image" src="https://user-images.githubusercontent.com/37161315/157011988-835f0ce9-9ad1-409b-9ebe-77a1c725bf35.png">

Reply:
Now remove.